portfolio/src/components/Card.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Styled from 'styled-components';
+import theme from '../styles/theme.ts';
+import { space, SpaceProps } from 'styled-system';
+
+const Container = Styled.div<SpaceProps>`
+    padding: 20px;
+    background-color: ${theme.foreground};
+    border-radius: 12px;
+    min-width: 180px;
+    margin-bottom: 30px;
+
+    ${space}
+`;
+
+type Props = {
+  children: React.ReactChild | Array<React.ReactChild>;
+  mb: SpaceProps.mb;
+};
+
+const Card = ({ children, mb }: {}) => {
+  return <Container {...{ mb }}>{children}</Container>;
```

Comment:
Don't understand this

Reply:
`mb` is a variable that means `margin-bottom` - its just something the library styled-system gives to make it easy to add margin as a variable on a component. The `...{ mb }` is a short way to send a variable to a component 

I could have done this: 
```jsx
   <Container mb={mb} />
```

but I chose to do it that way just in case I wanted to add more properties later like this:
```jsx
   <Container {...{ mb, ml, mr }}/>
```